woocommerce/woocommerce-gutenberg-products-block
# Add link to PRs merged in the last day to nightly release

## What
From recent internal discussions (p1697028572525809-slack-C02UBB1EPEF), we have decided it would be useful to include a link to Github PRs that were merged in the last day, in the nightly release. This PR takes care of this

## Why
To ain in debugging failed tests that use the nightly release for teams like SomewhereWarm


## Testing Instructions



This is a scheduled GH action that runs every night at midnight. So the only way to check it is to wait until the next Nightly Build action runs and check that the body of the release includes a link to GH PRs that were merged in the last 24 hours. Specifically, please check the date on the search filter is correct


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:
